src/content/docs/guides/configure-biome.mdx

```diff
@@ -187,33 +178,36 @@ Some shells don't support the recursive glob `**`.
 :::
 
 The Biome configuration file can be used to refine which files are processed.
-You can explicitly list the files to be processed using `include` and the files not to be processed using `ignore`.
-`include` and `ignore` accepts globs patterns such as `src/**/*.js`.
-See the [related section](#glob-syntax-explained) for which glob syntaxes are supported.
-`include` is always applied first before applying `ignore`.
-This allows you to include some files and to ignore some of the file you included.
-
-:::note
-`include` and `ignore` have a slightly different semantics.
-`include` doesn't prevent Biome of traversing a folder.
-This means that if you want to prevent Biome from traversing a folder, you have to add the folder to `ignore`.
+You can explicitly list the files to be processed using
+[the `files.includes` field](/reference/configuration). `files.includes` accepts
+[glob patterns](/reference/configuration#glob-syntax-reference) such as
+`src/**/*.js`. Negated patterns starting with `!` can be used to exclude files.
+
+Paths and globs inside Biome's configuration file are always resolved relative
+to the folder the configuration file is in.
+
+`files.includes` applies to all of Biome's tools, meaning the files specified
+here are processed by the linter, the formatter and the assist, unless specified
+otherwise. For the individual tools, you can further refine the matching files
+using `<tool>.includes`.
+
+:::important
+The global `files.includes` has slightly different semantics than the more
+specific `<tool>.includes` fields. Any file or folder that doesn't match
+`files.includes` is excluded from use by any of Biome's tools. This means that
+any tool-specific `includes` field can never match a file that doesn't also
+match `files.includes`.
```

Comment:
Actually it is even a bit more subtle.
Folders are only matched against negated patterns because folders like `src` doesn't match a pattern like `src/file.js`.

```suggestion
specific `<tool>.includes` fields. Any file that doesn't match
`files.includes` is excluded from use by any of Biome's tools. This means that
any tool-specific `includes` field can never match a file that doesn't also
match `files.includes`.
Also, Biome will not traverse a folder that matches a negated pattern from `files.includes`.
```

Reply:
Thanks! I don't think I would change the description though: After all, _not traversing_ a folder causes all the files in that folder to not match. So the original wording already implies the explanation you're giving. But while your explanation does help to explain _why_ those files don't match, it is actually harder to understand for the reader because you're overwhelming them with technical details that don't really matter for the end result.